packages/vue-searchbox/examples/with-vue-table/src/App.vue

```diff
@@ -0,0 +1,169 @@
+              field: 'original_title.search',
+              weight: 3
+            }
+          ]"
+          title="Search"
+          placeholder="Search for Books"
+          :autosuggest="true"
+          :defaultSuggestions="[
+            {
+              label: 'Songwriting',
+              value: 'Songwriting'
+            },
+            {
+              label: 'Musicians',
+              value: 'Musicians'
+            }
+          ]"
+          :size="10"
+          :debounce="100"
+          queryFormat="or"
+          fuzziness="AUTO"
+          :showClear="true"
+          :showVoiceSearch="true"
+          :URLParams="true"
+          class="custom-class"
+          :enableQuerySuggestions="true"
+          iconPosition="left"
+        />
+        <div class="row">
+          <div>
+            <search-component
+              id="result-component"
+              :dataField="['original_title']"
+              :size="5"
+              :react="{ and: ['search-component', 'author-filter'] }"
+            >
+              <div
+                class="result-list-container"
+                slot-scope="{ loading, error, results, size, from, setFrom }"
+              >
+                <div v-if="loading">Fetching Results ....</div>
+                <div v-if="Boolean(error)">
+                  Something went wrong! Error details
+                  {{ JSON.stringify(error) }}
+                </div>
+                <p v-if="!loading && !error">
+                  {{ results.numberOfResults }} results found in
+                  {{ results.time }}ms
+                </p>
+                <div v-if="!loading && !error">
+                  <v-client-table :data="results.data" :columns="columns" :options="options"></v-client-table>
+                  <div class="pagination-wrapper">
+                    <paginate
+                      v-if="results.numberOfResults"
+                      :value="Math.ceil(from / size) + 1"
+                      :page-count="Math.ceil(results.numberOfResults / size)"
+                      :click-handler="page => setFrom((page - 1) * size)"
+                      :prev-text="'Prev'"
+                      :next-text="'Next'"
+                      :container-class="'pagination'"
+                    ></paginate>
+                  </div>
+                </div>
+              </div>
+            </search-component>
+          </div>
+        </div>
+      </div>
+    </search-base>
+  </div>
+</template>
+
+<script>
+import Paginate from "vuejs-paginate";
+import {
+  SearchBase,
+  SearchComponent,
+  SearchBox
+} from "@appbaseio/vue-searchbox";
+import "./styles.css";
+
+export default {
+  name: "app",
+  components: {
+    SearchBase,
+    SearchBox,
+    SearchComponent,
+    Paginate
+  },
+  data() {
+    return {
+      columns: [
+        "title",
+        "authors",
+        "average_rating",
+        "language_code",
+        "isbn",
+        "ratings_count"
+      ],
+      options: {
```

Comment:
@nikhil-varma Are you using it?

Reply:
It's the same code which you had in the demo.. So I just focused on the table rendering...